app/controllers/application_controller.rb

```diff
@@ -27,4 +29,14 @@ def error(status_code, exception = nil)
   def set_expiry(age = 60.minutes)
     expires_in age, public: true unless Rails.env.development?
   end
+
+  def set_cors_headers
+    headers['Access-Control-Allow-Origin'] = '*'
+    headers['Access-Control-Allow-Methods'] = 'GET'
+    headers['Access-Control-Allow-Headers'] = 'Origin, X-Requested-With, Content-Type, Accept'
```

Comment:
`Access-Control-Allow-Methods` and `Access-Control-Allow-Headers` are [used in response to a preflight request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Allow-Methods), but we don't support `OPTIONS` requests at the moment so preflighting won't work. Since Calendars [only supports `GET` requests](https://github.com/alphagov/calendars/blob/master/config/routes.rb) I suspect [we don't need to support preflighting](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Simple_requests), so wouldn't need to set these headers either.